modules/standard/Random.chpl

```diff
@@ -603,6 +778,7 @@ module Random {
 
       /*
         Advances/rewinds the stream to the `n`-th value in the sequence.
+        The first value is with n=1.
```

Comment:
This wording reads awkwardly to me.  "is with" -> "corresponds to"?

The following comment says n must be non-negative.  That leaves a vagueness as to what happens when n is zero.